util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java

```diff
@@ -82,16 +83,47 @@ public DefaultSharedIndexInformer(
         new DeltaFIFO(
             (Function<KubernetesObject, String>) cache.getKeyFunc(),
             (Cache<KubernetesObject>) cache),
-        cache);
+        cache,
+        null);
   }
 
-  public DefaultSharedIndexInformer(
+
+    public DefaultSharedIndexInformer(
+        Class<ApiType> apiTypeClass,
+        ListerWatcher<ApiType, ApiListType> listerWatcher,
+        long resyncPeriod,
+        Cache<ApiType> cache,
+        BiConsumer<Class<ApiType>, Throwable> exceptionHandler) {
+        this(apiTypeClass, listerWatcher, resyncPeriod,
+            // down-casting should be safe here because one delta FIFO instance only serves one
+            // resource
+            // type
```

Comment:
nit pull this comment up to the previous line.

Reply:
Fixed.